setup.py

```diff
@@ -40,6 +40,7 @@
     entry_points={
         "prefect.collections": [
             "prefect_azure = prefect_azure",
+            "prefect_azure_container_instance = prefect_azure.container_instance",
```

Comment:
The container instance block is importable from the top level, so this extra entry point isn't necessary.
```suggestion
```

Reply:
This one is done, but GitHub doesn't realize it. :)